src/server/CMakeLists.txt

```diff
@@ -4,10 +4,11 @@ cxx_link(dragonfly base dragonfly_lib)
 if (CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64" AND CMAKE_BUILD_TYPE STREQUAL "Release")
   # Add core2 only to this file, thus avoiding instructions in this object file that
   # can cause SIGILL.
-  set_source_files_properties(dfly_main.cc PROPERTIES COMPILE_FLAGS -march=core2 COMPILE_DEFINITIONS
-                              SOURCE_PATH_FROM_BUILD_ENV=${CMAKE_SOURCE_DIR})
+  set_source_files_properties(dfly_main.cc PROPERTIES COMPILE_FLAGS -march=core2)
```

Comment:
i pulled it out from the "if" block.

Reply:
ohhh, I think the mistake was the `assignment`,

maybe this would work:

```
set_source_files_properties(dfly_main.cc PROPERTIES COMPILE_FLAGS -march=core2 COMPILE_DEFINITIONS SOURCE_PATH_FROM_BUILD_ENV ${CMAKE_SOURCE_DIR}) 
```

If this is the case, the syntax was wrong. Plz confirm and I will accept, I am fine with the two step chore with `set_property` below

Reply:
ohhh I am an idiot, NVM